courses/machine_learning/deepdive2/text_classification/solutions/rnn_encoder_decoder.ipynb

```diff
@@ -971,7 +971,7 @@
    "cell_type": "markdown",
```

Comment:
<p>same as above, </p><p>I think this reads better as </p><p>"Let's now average the&nbsp;<code>bleu_1</code>&nbsp;and&nbsp;<code>bleu_4</code>&nbsp;scores for all the sentence pairs in the eval set.&nbsp;The next cell takes some time to run, the bulk of which is decoding the 6000 sentences in the validation set. Please wait unitl completes."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
Hi @munnm ,

Updated the instruction as per you suggestion.

Thanks!